### ISSUE TITLE
Which yaml file to use for TA (Wide)

### ISSUE THREAD

**Issue by zhengyu998** [automl/trivialaugment#3]
Hello! Which config files did you use to train WRN-28-10 on CIFAR10/100 for TA (Wide) with and without augmented batch?

**Comment by SamuelGabriel** [comment]
Hey, good point. There are two versions of the paper: in the first we focused on the RA search space, in the second we focus on the wide search space. I did not update the configs after updating the paper. I will add the relevant configs, for now you can see:  https://github.com/automl/trivialaugment/blob/master/confs/wresnet28x10_cifar10_b128_maxlr.1_weighteduniaug_w%3D0%2C1_widesesp_nowarmup_8timesbatch_200epochs.yaml

To make it a standard training you can remove `all_workers_use_the_same_batches` and reduce the number of GPUs to 1.
To make it cifar100, replace cifar10 with cifar100.

**Comment by xiaobai-marker** [comment]
Hello!the following code is used directly: 
transforms.Compose([transforms.TrivialAugmentWide(),
                                   transforms.RandomResizedCrop(224),
                                   transforms.RandomHorizontalFlip(),
                                   transforms. ToTensor(),
                                   transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])]), 
after I found that both validation loss and training loss increased on mobilev2 and resnet50 instead, am I using the wrong method, the training set is the flower dataset

**Comment by xiaobai-marker** [comment]
Hello!the following code is used directly:
`transforms.Compose([transforms.TrivialAugmentWide(),
transforms.RandomResizedCrop(224),
transforms.RandomHorizontalFlip(),
transforms. ToTensor(),
transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])])`
after I found that both validation loss and training loss increased on mobilev2 and resnet50 instead, am I using the wrong method, the training set is the flower dataset

**Comment by SamuelGabriel** [comment]
So, I updated the configs now to include `wide` variants for all experiments in the main table (2).
